ibtk/src/math/HierarchyMathOps.cpp

```diff
@@ -1987,7 +1987,7 @@ HierarchyMathOps::interp(const int dst_idx,
                          const bool src_cf_bdry_synch)
 {
     if (src_ghost_fill) src_ghost_fill->fillData(src_ghost_fill_time);
-    Pointer<NodeDataFactory<NDIM, double> > data_factory = dst_var->getPatchDataFactory();
+    Pointer<CellDataFactory<NDIM, double> > data_factory = dst_var->getPatchDataFactory();
```

Comment:
I fixed another copy-paste error.

Reply:
Gotta get rid of the silently failing implicit conversion facilitated by `Pointer`.